### PR TITLE
fix(ui): link to noderun: missing name query params

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: ui/src/app/shared/workflow/wnode/pipeline/wnode.pipeline.component.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class WorkflowWNodePipelineComponent {
     displayLogs() {
         if (this.noderun) {
             this._router.navigate(['node', this.noderun.id], {
-                relativeTo: this._activatedRoute,
+                relativeTo: this._activatedRoute, queryParams: { name: this.node.name }
```
Comment (richardlt) [inline ui/src/app/shared/workflow/wnode/pipeline/wnode.pipeline.component.ts:39]: Looks like this data should be retreive directly from the noderun, what will happen is I set manually this data in the url with random data?

File: ui/src/app/shared/workflow/wnode/wnode.component.ts [hunk 2]
```diff
@@ -190,6 +190,7 @@ export class WorkflowWNodeComponent implements OnInit {
             case 'logs':
                 this._router.navigate(['node', this.currentNodeRun.id], {
                     relativeTo: this._activatedRoute,
+                    queryParams: { name: this.node.name }
```
Comment (richardlt) [inline ui/src/app/shared/workflow/wnode/wnode.component.ts:193]: Same

File: ui/src/app/shared/workflow/wnode/pipeline/wnode.pipeline.component.ts (hunk 1 above)
Comment (sguiheux) [reply ui/src/app/shared/workflow/wnode/pipeline/wnode.pipeline.component.ts:39]: This is just to have a nice title in browser tabs.
The process that set the title is located in app.component.ts and just analyze the url.

Comment (richardlt) [reply ui/src/app/shared/workflow/wnode/pipeline/wnode.pipeline.component.ts:39]: Mokay

**Pull request merged**